src/api/aux/utils.ts

```diff
@@ -14,7 +14,10 @@ export function blobToByteArray(blob: Blob): Promise<Uint8Array | null> {
       }
 
       if (typeof reader.result === 'string') {
-        const b64 = reader.result.split('data:application/octet-stream;base64,')[1]
+        const b64 = reader.result.split('base64,')[1]
+        if (!b64) {
+          reject({ reader })
+        }
 
         resolve(toByteArray(b64))
```

Comment:
`resolve(toByteArray(b64))` would be invoked even if `reject({ reader })` was called, if I'm not mistaken. Do you agree?

Reply:
It's not true. Reject & Resolve stops a call